frontends/PyCDE/python/requirements.txt

```diff
@@ -0,0 +1,10 @@
+setuptools>=42
+setuptools_scm>=6.2
+wheel
+cmake>=3.12
```

Comment:
this line and above are not required -- they are only in pyproject.toml since they are needed to build the python wheel.

Reply:
Removed!